INSTALL.md

```diff
@@ -0,0 +1,202 @@
+# Installation instructions
+
+Full installation instructions may be found on the
+[`oomph-lib` homepage](http://www.oomph-lib.org). There are two main methods for
+installing `oomph-lib`:
+
+**1. Installation using `autogen.sh`**
+
+   `oomph-lib`s top-level directory contains a shell script [autogen.sh](autogen.sh)
+   which guides the user through the installation procedure and suggests
+   sensible defaults for various configure/installation options.
+   We highly recommend using this script to perform the installation.
+
+**2. Installation using the standard GNU build process**
+
+   You may, of course, also use GNU's standard
+
+      configure && make && make install && make check
```

Comment:
Backticks

Reply:
An indented block *does* have the same effect, but I agree it's probably best to add them in. I'll take care of it. Thanks for the spot!